### DISCUSSION TITLE
Meeting Agenda Wed 15th March

### DISCUSSION THREAD

**Discussion by ColinEberhardt** [finos/technical-oversight-committee#57] [Meeting Agenda]
Hi All,

This is our draft agenda for our next TSC meeting. Items to discuss include:

- TSC governance and charter - @ColinEberhardt 
- Discuss our role in approving SIG formation, see for example https://github.com/finos/community/issues/226
- @opoupeney previously proposed running a session to map the FINOS catalogue of projects to different business domains/areas (Interop, Reg, Workflows, AI, Visualization, ...) - is there appetite for such a discussion?
- @eminty69 any update on the project archiving work? https://github.com/finos/community/issues/203

**Comment by ColinEberhardt** [comment]
We should consider moving the meeting agenda into GitHub issues, as this is more standard across FINOS projects, see https://github.com/finos/technical-steering-committee/pulls

**Comment by ColinEberhardt** [comment]
We've also been asked by @TheJuanAndOnly99 to participate on project activations. I've [added the two projects to our board](https://github.com/orgs/finos/projects/39), as this is something we haven't done before, we should discuss what this entails.

**Reply by mindthegab** [reply]
Here's the current law of the land, aka checklist for activation: https://community.finos.org/docs/governance/Software-Projects/stages/active - and projects provide evidence on the issue - so I think this should be a fairly straightforward verification and TSC having discretion on the final decision in case some of the checklist items are not fully matched.

**Comment by maoo** [comment]
Thanks @ColinEberhardt for circulating the agenda! I've few additions to suggest:
- Briefly discuss how we can streamline activity around new contributions and activation requests, as you suggested in your last comment
- FINOS Hackathon at BMO (NYC, 3/4 May) - we're looking for sample use cases around Desktop Interoperability and Open RegTech; I'd also like to have the opportunity to ask TSC members to a) share the [landing and registration page](https://www.finos.org/hosted-events/2023-05-03-member-exclusive-hackathon-bmo) within their firms and b) volunteer as subject matter experts and judges

**Comment by mindthegab** [comment]
> * TSC governance and charter - @ColinEberhardt

@ColinEberhardt I might be able to join only the latter 30m of the meeting, so can I ask this agenda item to be the last one? /cc @jgavronsky @maoo
